.changeset/spicy-schools-sparkle.md

```diff
@@ -2,4 +2,8 @@
 "@kaizen/components": major
 ---
 
-Changes the icon API to require the role prop, then use a discriminated union to require a label when role="img"
+Changes the props API on KAIO icon components to better ensure accessibility:
+- `role` is now a required prop, prompting engineers to consciously choose between meaningful (`role="img"`) or decorative (`role="presentation"`) on each usage. Previous default was `role="img"`.
+- The `title` prop has been renamed to `aria-label` so that there's less confusion with the native `title` HTML attribute, and to be in line with how this is implemented.
+- `aria-label` is now required when the icon is meaningful (`role="img"`). Previously this would result in a console warning.
+- Removes the ability to provide a `desc`. We can't see any reasons why we'd need to provide a description for an icon, and browser/screen reader support may be patchy anyway. If you run into any usages of `desc`, check if there's anything useful in there that's not already in the `aria-label`. If so, just copy it into the `aria-label`.
```

Comment:
Because technically this is a new component in a new package, I would actually write this as if it's a new component, so consider this a feature not a major breaking change.

Reply:
Alright, you still want a major bump here though right?

Reply:
Nah, `minor` -- we're already making consumers change their code when they migrate to KAIO, so having to update the API at the same time is reasonable.